auth_menus/__manifest__.py

```diff
@@ -2,7 +2,7 @@
 # See LICENSE file for full copyright and licensing details.
 {
     'name': "Menus for Authentication Modules",
-    'version': '18.0.4.0.0',
+    'version': '19.0.4.0.0',
```

Comment:
We bump this on next major version, to indicate the evolution of a module (meaning its a fifth iteration).
```suggestion
    'version': '19.0.5.0.0',
```